### DISCUSSION TITLE
Reusing part of the input  from title when creating the  meeting note (or  a way to pick the custom date and reuse it later)

### DISCUSSION THREAD

**Discussion by danieltomasz** [SilentVoid13/Templater#562] [Help]
I am using templater to create meeting notes, I am looking for a way either to dynamically input a date when creating a note or just get a first item (this is date that I  get by expand using espanso) from what I  am  inputting (current approach with<% tp.file.title.split(" ")[0] %>  doesn't work becasue not doesnt exits yet) Note titles has format "YYYY-MM-DD Person"

Maybe  asking "date" was a misdirection - I will generalize my question - what I need is to reuse part of my input (user input) when creating note later in filling the template (it may be for example first of second word) 
```
<%*
let qcFileName= await tp. system.prompt("Note Title")
titleName = qcFileName 
await tp.file.rename(titleName)
await tp.file.move ("/Communication/Meetings/"+ titleName);
-%>
---
title: "<% qcFileName %>"
author: 
date:  <% tp.file.title.split(" ")[0] %>
tags: [upnext,metting]
project:
---

**Comment by AB1908** [comment]
Try `<% tp.date.now() %>`. See docs: https://silentvoid13.github.io/Templater/internal-functions/internal-modules/date-module.html#tpdatenowformat-string--yyyy-mm-dd-offset-numberstring-reference-string-reference_format-string

**Reply by danieltomasz** [reply]
The date might be as well in  the future (lets say I am planning meeting in advance), what I need is to use the date I will input into many files later
Maybe "date" was a misdirection - I will generalize it - what I need is to reuse part of my input (user input) when creating note later in filling the template (it may be for example first of second word)

**Reply by AB1908** [reply]
For a prompt, can use `tp.system.prompt` and pass that as input to `create_new`.
